client/post-editor/editor-discussion/index.jsx

```diff
@@ -39,7 +44,7 @@ export class EditorDiscussion extends React.Component {
 		translate: identity,
 	};
 
-	getDiscussionSetting = () => {
+	getDiscussionSetting() {
```

Comment:
do you mainly make these changes from autobound to regular functions for the performance benefit?